src/Panel.tsx

```diff
@@ -33,23 +33,37 @@ function liveRegionChangesReducer(state: State, action: Action) {
       };
     case "clear":
       return INITIAL_STATE;
+    // @ts-expect-error
+    case "unknown_type":
+      return state;
     default:
       return state;
   }
 }
 
+type KnownEventTypes = keyof typeof EVENTS | typeof STORY_CHANGED;
+type EventHandlers = Record<
+  KnownEventTypes,
+  (payload?: ChangeOptions | void) => void
+>;
+
 export const Panel: React.FC<PanelProps> = (props) => {
   const [ariaLiveRegionChanges, dispatchAriaLiveRegionCange] = React.useReducer(
     liveRegionChangesReducer,
     INITIAL_STATE
   );
 
   // https://storybook.js.org/docs/react/addons/addons-api#usechannel
-  const emit = useChannel({
+
+  const handlers: EventHandlers = {
     [STORY_CHANGED]: () => dispatchAriaLiveRegionCange({ type: "clear" }),
-    [EVENTS.ADD_CHANGE]: (change) =>
-      dispatchAriaLiveRegionCange({ type: "add_change", payload: change }),
-  });
+    [EVENTS.ADD_CHANGE]: (payload: ChangeOptions) =>
+      dispatchAriaLiveRegionCange({ type: "add_change", payload }),
+    // @ts-expect-error
+    iWillErrorBanana: () => {},
```

Comment:
I'm very interested in this @ts-expect-error annotation.

I assume that it is a more specific @ts-ignore, where we're saying "I know that this might result in an error and I'm ok with that. But also, tell continue reporting any other TS errors that I missed"

Is this assumption correct?

Reply:
Yup, spot on. It's typically used in tests, which is a whole other conversation 😅

Reply:
we'll have to have that conversation over lunch some time :)